convention.md

```diff
@@ -88,11 +88,12 @@ needs this, then it should provide for an escape mechanism on application level.
  
 #### Color
 
-- Color payload validity varies depending on the property format definition of either "rgb" or "hsv"
+- Color payload validity varies depending on the property format definition of either "rgb", "hsv" or "xyz"
 - Both payload types contain comma separated numbers of differing restricted ranges. The numbers must conform to the [float](#float) format
 - The encoded string may only contain the [float](#float) numbers and the comma character ",", no other characters are permitted, including spaces (" ")
 - Payloads for type "rgb" contain 3 comma separated values of [floats](#float) with a valid range between 0 and 255 (inclusive). e.g. 100,100,100
 - Payloads for type "hsv" contain 3 comma separated values of [floats](#float). The first number has a range of 0 to 360 (inclusive), the second and third numbers have a range of 0 to 100 (inclusive).  e.g. 300,50,75
+- Payloads for type "xyz" contain 2 comma separated values of [floats](#float) with a valid range between 0 and 1 (inclusive). The z value can be calucated via z=1-x-y and is therefore not transmitted. (see [CIE_1931_color_space](https://en.wikipedia.org/wiki/CIE_1931_color_space))
```

Comment:
Is it wise to have 3 color systems? This forces every controller to support all 3 of them, and do all conversions. Wouldn't it make more sense to support 1 only, and have the conversion logic only where it is needed?

Reply:
that could mean device implementers doing this, so in hind sight I think doing it on the controller side is fine.